rank/scripts/rank.ts

```diff
@@ -0,0 +1,49 @@
+import { Index, QueryEnv, queryEnvs } from '../types/searchTemplate'
+import { code, gatherArgs, info } from './utils'
+
+import { exec } from 'child_process'
+import { listIndices } from '../services/search-templates'
+import { tests as possibleTests } from '../data/tests'
+
+global.fetch = require('node-fetch')
+
+async function go() {
+  const indices: Index[] = await listIndices()
+
+  const possibleTestIds = Object.values(possibleTests)
+    .flatMap((testSet) => testSet.map((test) => test.id))
+    .filter((v, i, a) => a.indexOf(v) == i) // keep unique
```

Comment:
What are these values? Why does filtering them keep them unique?

Reply:
longer comment added in https://github.com/wellcomecollection/catalogue-api/pull/310/commits/e6abd8e59aaa92d686c2afa1783fe4b6faebbe4d !